packages/driver/src/multi-domain/cypress.ts

```diff
@@ -67,6 +67,22 @@ const setup = () => {
   handleSocketEvents(Cypress)
   handleSpecWindowEvents(cy)
 
+  let beforeScreenshotCallback
+
+  Cypress.on('before:screenshot', (config, cb) => {
+    beforeScreenshotCallback = cb
+
+    specBridgeCommunicator.toPrimary('before:screenshot', config)
+  })
+
+  specBridgeCommunicator.on('before:screenshot:end', () => {
+    beforeScreenshotCallback()
+  })
+
+  Cypress.on('after:screenshot', (config) => {
+    specBridgeCommunicator.toPrimary('after:screenshot', config)
+  })
+
```

Comment:
Let's go ahead and move this into its own file `multi-domain/screenshots.ts` with a `handleScreenshots` function.